src/components/DefaultRedirectionSection.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import {
+  useQuery,
+  hasQueryBeenLoaded,
+  deconstructRedirectLink
+} from 'cozy-client'
+
+import Select from 'components/Select'
+import { buildAppsQuery } from 'lib/queries'
+import { useWebviewIntent } from 'cozy-intent'
+
+const EXCLUDED_SLUGS = ['settings', 'store']
+
+const formatOptions = (appsResult, t) => {
+  if (!hasQueryBeenLoaded(appsResult)) return []
+
+  return appsResult.data
+    .filter(({ slug }) => !EXCLUDED_SLUGS.includes(slug))
+    .map(({ slug, name }, index) => {
+      if (slug === 'home') {
+        return {
+          index: -1, // to set 'home' before everything else when sorting
```

Comment:
Two questions:
- Should we move this logic in the `.sort()` instead? Using a .map to do a sort is a bit odd
- Don't we sort other apps alphabetically?

Reply:
After reading a bit more, I understand that we do the map in order to rename the `home` option, so it is more optimal to also do the index thing here.
But the question still remain about the alpabetical sort

Reply:
When I checked they are returned alphabetically sorted by the stack so I was counting on it.

Reply:
Ok.

@Crash-- do you know if this is a "lucky" effect or if we can consider this to be the norm?

However this is a detail that should not block the PR so I will approve.